src/react/components/_academics/AcademicSummary/SectionIncompleteGradingStatus.js

```diff
@@ -46,7 +46,7 @@ SectionIncompleteGradingStatus.propTypes = propTypes;
 
 const mapStateToProps = ({ myAcademics }) => {
   const {
-    studentLinks: { academicGuideGradesPolicy: academicGuideGradesPolicyLink }
+    studentLinks: { academicGuideGradesPolicy: academicGuideGradesPolicyLink } = {}
```

Comment:
Didn't you extract an advisor link here when we were working on this together?

Reply:
@developish - Yes. Then I decided against it. I went back and evaluated the logic of the `:advisorLinks` object, and made sure that the code would prefer those links over the `:studentLinks` object if present. Then I made `:studentLinks` available to Advisors as well so that this kind of issue won't keep cropping up in the future.

Reply:
Great, in that case it looks good to me